README.md

```diff
@@ -1,157 +1,179 @@
+   ```bash
+   $ sfx-rb-trace-bootstrap --install-deps <dep1>,<dep2>,<dep3>,<dep4>
+
+### Manually install the library
+
+1. Set the service name to identify the service in SignalFx:
+   ```bash
+   $ export SIGNALFX_SERVICE_NAME = "yourService"
+   ```
+2. If you're sending trace data to a SignalFx ingest endpoint or OpenTelemetry
+   Collector, set the access token for your organization:
+   ```bash
+   $ export SIGNALFX_ACCESS_TOKEN = "yourToken"
+   ```
+3. If you're sending trace data to an endpoint that's not `http://localhost:9080/v1/trace`,
+   set the endpoint:
+   ```bash
+   $ export SIGNALFX_ENDPOINT_URL = "yourEndpoint"
+   ```
+4. Download the [latest release](https://github.com/signalfx/signalfx-ruby-tracing/releases/latest) of the tracing library.
+5. Add `signalfx-tracing` to your application's Gemfile:
+   ```bash
+   $ gem 'signalfx-tracing'
+   ```
+6. Add each applicable instrumentation to your application's Gemfile. For
+   example, this is how you add Rails and Redis:
+   ```bash
+   $ gem 'rails-instrumentation'
+   $ gem 'redis-instrumentation'
+   ```
+   For more information about instrumentation names, see
+   [Supported libraries](#supported-libraries).
+7. Install the gems for the tracing library and instrumentations:
+   ```bash
+   $ bundle install
+   ```
+
+## Configure instrumentation for a Ruby application
+
+Configure the instrumentation anywhere in the setup portion of your code or
+before doing anything that needs to be traced.
+
+For example, with Rails, configure instrumentation in `config/initializer/tracing.rb`.
+
+You can configure instrumentation automatically or manually. Manual
+instrumentation is convenient when you want to trace only some libraries.
+
+### Automatically instrument code:
+
+Configure the auto-instrumentor to check for modules defined in the code and
+instrument them if available:
 
 ```ruby
-gem 'signalfx-tracing'
-```
-
-Then execute the following (or use your desired installation method for your application).
+require 'signalfx/tracing'
 
-```bash
-$ bundle install
+SignalFx::Tracing::Instrumenter.configure(auto_instrument:true)
 ```
 
-## Usage
-
-Configure the instrumentation anywhere in the setup portion of your code or before doing anything
-that needs to be traced.
-
-For Rails, this would go in `config/initializer/tracing.rb`.
+### Manually instrument code:
 
-The instrumentation can be done automatically, where the auto-instrumenter will
-check for modules defined in the code and instrument them if available:
+Specify which libraries to instrument:
 
 ```ruby
 require 'signalfx/tracing'
 
-SignalFx::Tracing::Instrumenter.configure(auto_instrument:true)
-```
-
-Manual configuration may be desirable when only some libraries should be traced.
-These instrumentations to can be selected in a block:
-
-```ruby
-SignalFx::Tracing::Instrumenter.configure do |patcher|
-    patcher.instrument(:LibName)
+SignalFx::Tracing::Instrumenter.configure do |p|
+    p.instrument(:LibName)
```

Comment:
Mind updating `:LibName` to something like `<:MyLibName>`?  Otherwise it seems like an actual symbol and not a placeholder.

Reply:
An example could also be helpful